x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/index.tsx

```diff
@@ -128,55 +127,34 @@ const getDescriptionItem = (
       indexPatterns,
     });
   } else if (field === 'threat') {
-    const threat: IMitreEnterpriseAttack[] = get(field, value).filter(
+    const threat: IMitreEnterpriseAttack[] = get(field, data).filter(
```

Comment:
This would be better as just an unsafe cast rather than lodash get for a refactor like so:

```ts
    // TODO: use io-ts or a TypeGuard "is" to correctly narrow this later.
    const threats = data as { threat: IMitreEnterpriseAttack[] };
    const threat = threats.threat.filter(singleThreat => singleThreat.tactic.name !== 'none');
```

It would be cleaner and if/when you have the io-ts type you can run a decode on it and handle a left and a right with the decode for more run time safety to do maybe a throw error if a maintainer messes it up.

Reply:
Deferred to https://github.com/elastic/kibana/issues/60567